File: ARCADE/Intro/Level 2_2: shapeArea.cs
    using System;
     
    public class Test
    {
    	public static int shapeArea(int n) {
        	if(n == 1) return 1;
        	else return(shapeArea(n-1) + (n-1)*4);
        }
     
    	public static void Main()
    	{
    		int n = 2;
    		Console.WriteLine(shapeArea(n));
    		int m = 3;
    		Console.WriteLine(shapeArea(m));
    	}
    }

File: ARCADE/Intro/Level 2_2: shapeArea.pas
    program shapeArea;
     
    function shapeArea(n: integer): integer;
    begin
      if(n = 1) then shapeArea := 1
      else shapeArea := shapeArea(n-1) + (n-1)*4;
    end;
     
    var n : integer;
    begin
    	n := 2;
    	writeln(shapeArea(n));
    	n := 3;
    	writeln(shapeArea(n));
    end.
